src/main/java/org/myrobotlab/service/Polly.java

```diff
@@ -400,6 +409,16 @@ public boolean isReady() {
     setReady(polly != null ? true : false);
     return ready;
   }
+  
+  @Override
+  public ServiceConfig apply(ServiceConfig c) {
+    // must initialize polly before getting voices
+    // must get voices before setVoice
+    getVoices();
+    // this will set voice based on config
+    super.apply(c);

```

Comment:
normally you should call the super apply method first unless there's a good reason not to...

Reply:
changed .. checked Polly seems ok .. additionally I added unit test using mockito which was a pain ...
https://github.com/MyRobotLab/myrobotlab/blob/config-with-peers-and-subscriptions/src/test/java/org/myrobotlab/service/PollyTest.java  .. mockito should be pulled in as a test dep, which i did in the pom